apps/public-docsite-v9/.storybook/fix-title.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require('path');
+
+try {
+  const args = process.argv.slice(2);
+  const [title, distPath] = args;
+
+  console.log(`Rewriting index.html document title to ${title}.`);
+
+  const filePath = `${distPath}/storybook/index.html`;
+  const document = fs.readFileSync(path.resolve(__dirname, filePath), 'utf8');
```

Comment:
**suggestion:**

we can extract path to const to make it more dry

```js
const htmlDocumentPath = path.resolve(__dirname, filePath);
const htmlDocument = fs.readFileSync(htmlDocumentPath,'utf-8');
const updatedHtmlDocument = htmlDocument.replace(/<title>.*<\/title>/, `<title>${title}</title>`);

fs.writeFileSync(updatedHtmlDocument, output);
```

Reply:
Refactored the code to be less repetitive and also added an `iframe.html` title update since I forgot about that one and it was also having the "Webpack App" value.